packages/api/src/controllers/stream.js

```diff
@@ -62,7 +61,7 @@ function isActuallyNotActive(stream) {
 
 function activeCleanupOne(stream) {
   if (isActuallyNotActive(stream)) {
-    setActiveToFalse(stream);
+    db.stream.setActiveToFalse(stream);
```

Comment:
`await` here?

Reply:
We don't need result of this function and we don't want to wait for it to complete - it can execute in parallel.